File: Assignment 26/Program3.java
import java.util.*;

class Pattern
{
    public void Display(int iNo)
    {
        int iCnt=0,iCount=1;

        for(iCnt=1;iCnt<=(iNo*2);iCnt++)
        {
            if((iCnt%2)==0)
            {
                System.out.print("*\t");
            }
            else
            {
                System.out.print(iCount+"\t");
                iCount++;
            }
        }

    }
}
class Program3
{
    public static void main(String A[])
    {
        int iValue=0;

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the number:");

        iValue=sobj.nextInt();

        Pattern pobj=new Pattern();
        pobj.Display(iValue);
    }
}

File: Assignment 2/Program3.c
#include<stdio.h>

void Display(int iNo)
{
    if(iNo<10)
    {
        printf("Hello");
    }
    else
    {
        printf("Demo");
    }

}

int main()
{
    int iValue=0;

    printf("Enter The number:");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;

}